data/mods/Magiclysm/Spells/stormshaper.json

```
@@ -430,5 +431,461 @@
+    "spell_class": "STORMSHAPER",
+    "energy_source": "MANA",
+    "difficulty": 4,
+    "base_casting_time": 3000,
+    "final_casting_time": 1000,
+    "casting_time_increment": -80,
+    "base_energy_cost": 400,
+    "final_energy_cost": 200,
+    "energy_increment": -8,
+    "max_level": 25,
+    "min_duration": 57600,
+    "max_duration": 1440000,
+    "duration_increment": 57600
+  },
+  {
+    "id": "electric_waves",
+    "type": "SPELL",
+    "name": "Electric Waves",
+    "description": "Your hands start to emit a huge amount of energy, that you release with each your attack.  The damage is increases the more you master the spell.",
+    "valid_targets": [ "self" ],
+    "effect": "attack",
+    "effect_str": "electric_waves",
+    "shape": "blast",
+    "flags": [ "NO_LEGS", "SILENT", "NO_EXPLOSION_SFX" ],
+    "skill": "deduction",
+    "spell_class": "STORMSHAPER",
+    "energy_source": "MANA",
+    "difficulty": 5,
+    "base_casting_time": 3000,
+    "final_casting_time": 1000,
+    "casting_time_increment": -80,
+    "base_energy_cost": 400,
+    "final_energy_cost": 200,
+    "energy_increment": -8,
+    "max_level": 25,
+    "min_duration": 14400,
+    "max_duration": 360000,
+    "duration_increment": 14400
+  },
+  {
+    "id": "electric_waves_damage",
+    "type": "SPELL",
+    "name": { "str": "Electric Waves Damage" },
+    "description": "Blasts, generated by Electric Waves .",
+    "valid_targets": [ "ally", "hostile", "ground" ],
+    "effect": "attack",
+    "shape": "blast",
+    "min_damage": 10,
+    "max_damage": 35,
+    "damage_type": "electric",
+    "flags": [ "LOUD", "RANDOM_DURATION", "RANDOM_AOE", "RANDOM_DAMAGE" ],
+    "min_duration": 50,
+    "max_duration": 200,
+    "min_aoe": 1,
+    "max_aoe": 2,
+    "field_id": "fd_electricity",
+    "field_chance": 2,
+    "min_field_intensity": 1,
+    "max_field_intensity": 4,
+    "field_intensity_variance": 10
+  },
+  {
+    "id": "electric_arts",
+    "type": "SPELL",
+    "name": "Electric Arts",
+    "description": "This spell were designed for scholars, that just started to learn stormshaping.  It allows them to use their abilities, when their magic control is weak, and they can injure themselves with electricity. The effect grows with each level.",
+    "valid_targets": [ "self" ],
+    "effect": "attack",
+    "effect_str": "electric_arts",
+    "shape": "blast",
+    "flags": [ "NO_EXPLOSION_SFX" ],
+    "spell_class": "STORMSHAPER",
+    "energy_source": "MANA",
+    "difficulty": 1,
+    "base_casting_time": 48000,
+    "base_energy_cost": 400,
+    "final_energy_cost": 200,
+    "energy_increment": -8,
+    "max_level": 25,
+    "min_duration": 230400,
+    "max_duration": 5760000,
+    "duration_increment": 230400
+  },
+  {
+    "id": "summon_teravolt",
+    "type": "SPELL",
+    "name": "Summon Terravolt",
+    "description": "Summon a Teravolt, a small, semi-intelligent spirit, that home hostile creatures, and explode with huge spark cloud.",
```

```suggestion
    "name": "Summon Terravolt",
    "description": "Summon a Teravolt, a small, semi-intelligent spirit, that homes onto hostile creatures before exploding into a huge spark cloud.",
```
Is this supposed to be Terravolt or Teravolt 🤔

---

whoops, it must be Teravolt, like "Tera" - trillion, and "volt" - volt. Earthshaper stuff were merged already XD